### ISSUE TITLE
removing "virtual" prefix from entity_id

### ISSUE THREAD

**Issue by 50494554524F** [twrecked/hass-virtual#11]
how do i remove this prefix?! can you tell me the line of code to modify?! thanks

**Comment by twrecked** [comment]
Good question, I don't add the virtual, it's added by Home Assistant. You might be able to remove it by clearing the component domain.

**Comment by 50494554524F** [comment]
unfortunately you confirmed my idea, i tried to read the code (not an expert)... ok, not a big problem, you have to remember to change the names with real devices when done. thanks for your useful integration!

**Comment by twrecked** [comment]
And annoyingly, the fan device doesn't put `virtual` in front.

**Comment by twrecked** [comment]
Turns out I lied. I just needed to define a `name` property. Fixed in this PR #13 

It doesn't turn on by default, it would break too much of my own code,  but if you prefix a name with `!` it will omit the domain part. So this switch:

```yaml
- platform: virtual
  name: "!Test Switch"
  initial_value: 'on'
```

becomes `switch.test_switch`

**Comment by twrecked** [comment]
Fixed.
